src/Components/Footnote.js

```diff
@@ -23,6 +23,7 @@ const Footnote = (props) => {
     return (
         <details className="footnote" id={props.id} ref={detailsRef}>
             <summary>
+                <span className="visually-hidden">Footnote</span>
```

Comment:
For translating this, you can add just the English translation (the other ones will later be filled in by our translators on Weblate) to this file: https://github.com/datenanfragen/website/blob/master/src/i18n/en.json  
The top-level keys set the `scope` (I don't think any of the existing ones fit, so you have to invent a new one), the second-level keys the translation ID.

Then, you wrap the component in an `IntlProvider` and specify the translation's `scope` :

https://github.com/datenanfragen/website/blob/802a2805a57f54a7b344b74f15eba347298e3f7a/src/Components/Generator/TransportMediumChooser.js#L10

Finally, you can use `<Text>` with your translation ID:

https://github.com/datenanfragen/website/blob/802a2805a57f54a7b344b74f15eba347298e3f7a/src/Components/Generator/TransportMediumChooser.js#L12

Reply:
@baltpeter Thank you for the assistance! That helped a lot and this should be ready for another review.